datasets/fr/tresor/fr_tresor_gels_avoir.yml

```diff
@@ -56,6 +56,478 @@ assertions:
       Company: 60
 
 lookups:
+  sanction.program:
+    options:
+      - match:
+          - "ECOT2429026A du 29/10/2024 (Dispositif national de gel des avoirs - art L. 562-3 du code monétaire et financier)"
```

Comment:
Can we just do a regex check on `art L. 562-3 du code monétaire et financier` here?